components/log_parser.rb

```diff
@@ -17,8 +18,25 @@ def get_first_line
     @file_str[0]
   end
 
+  def get_players
+    @players = Set[]
```

Comment:
Qual a necessidade de declarar isso como uma variável de instância? Poderia ser uma variável normal.
E também, você poderia usar um array e no final usar o .uniq para tirar os duplicados. (caso não tenha considerado essa opção)

```suggestion
    players = Set[]
```

Reply:
Vi depois que podia fazer esse .uniq, vou alterar tambem.
Obrigado!